src/RNGestureHandlerModule.ts

```diff
@@ -2,4 +2,28 @@ import { NativeModules } from 'react-native';
 
 const { RNGestureHandlerModule } = NativeModules;
 
-export default RNGestureHandlerModule;
+type Directions = Readonly<{
+  RIGHT: 1;
+  LEFT: 2;
+  UP: 4;
+  DOWN: 8;
+}>;
```

Comment:
Can't it be an ENUM?

Reply:
IIRC enum semantics is different from an object. I'd like to keep possibly incompatible changes outside of this PR and maybe fix them in the follow-up.